apis/database.js

```diff
@@ -23,30 +25,60 @@ export function executeTransaction(statement, args = null) {
   });
 }
 
-export function createDatabases() {
+export async function createDatabases() {
+  await SecureStore.setItemAsync(DATABASE_VERSION_KEY, Constants.manifest.extra.localDatabaseVersion);
```

Comment:
Interesting. I think this value should be directly in the code and not from the configuration. It changes based on this one file and not anywhere else.

Reply:
Fair enough!